src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -31,9 +31,19 @@ namespace NUnit.Framework.Constraints
     /// item in a collection, succeeding only if a specified
     /// number of items succeed.
     /// </summary>
-    public class ExactCountConstraint : PrefixConstraint
+    public class ExactCountConstraint : Constraint
     {
-        private int expectedCount;
+        private readonly int expectedCount;
+        private readonly IConstraint itemConstraint;
```

Comment:
Good point since we haven't converted a lot of code yet, although I think they are just visual (not compile time) yellow lines in the code and I expect there are already many. @jnm2 can you confirm and maybe post a screenshot to refresh my memory?